modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-image-set-examples_{context}"]
+= Image set configuration examples
+
+// Is this section worthwhile?
+
+TODO
+
+// TODO intro
+// Providing a few examples to accomplish different things. Get help with which ones are most important. Only found one here (https://github.com/openshift/oc-mirror/tree/main/docs/examples) that used storageConfig - I assume we should only document ones that use it?
```

Comment:
@afflom @jpower432 I was considering adding a section with a few different image set configuration examples. The examples in https://github.com/openshift/oc-mirror/tree/main/docs/examples are all a lot smaller than the example we show in the section on creating the file: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

So I assume that means that a lot of that isn't required. So perhaps smaller examples for different use cases might be helpful to users. But I'd be curious to hear whether you think this is worthwhile or not. If not, no worries.

If we do think it's helpful, can either of you help with what examples would be helpful to show. When I look at that list of examples on GitHub, only one of them has "storageConfig". This goes along with my other question on whether "storageConfig" is going to be required. And if so, then I'd think that the examples that we show should all have that.